# api/errors/__init__.py

# https://fastapi.tiangolo.com/reference/fastapi/?h=exception_#fastapi.FastAPI.exception_handler
# from: https://fastapi.tiangolo.com/tutorial/handling-errors/

# end
